app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/terms.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxsubscriptioneligibilityfrontend.config.AppConfig
+@import uk.gov.hmrc.play.views.html.helpers.form
+
+@(postAction:Call)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main_template(title = messages("terms.title")) {
+
+    <h1 class="heading-large">@messages("terms.heading")</h1>
+
+    <div class="form-group">
+        <p>@messages("terms.line1")</p>
+            <ul class="bullets">
```

Comment:
```suggestion
            <ul class="list list-bullet">
```
This change will make the bullet points spread out more than they currently are, as per the prototype

Reply:
Also formatting of the file, ul tag should be tabbed to the same amount of the p tag